debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -572,4 +586,16 @@ private boolean isVariableScaleDecimal(final Column column) {
         return (column.scale() == 0 && column.length() == VARIABLE_SCALE_DECIMAL_LENGTH)
                 || (column.scale() == -1 && column.length() == -1);
     }
+
+    public static SpecialValue toSpecialValue(String value) {
+        switch (value) {
+        case N_A_N:
+            return SpecialValue.NaN;
+        case INF:
+            return SpecialValue.PositiveInfinity;
+        case "-" + INF:
```

Comment:
I'd suggest to extract another constant for that one.

Reply:
ok